### PR TITLE
[Hotfix] Fix the row keys to correctly render Bird's eye report table

### PR DESCRIPTION
**Pull request by PavelZurek**
## Change description

Fixes bug seen on this video:
[Screencast from 27.5.2024 14:24:57.webm](https://github.com/cortexapps/backstage-plugin/assets/32833929/14185326-6641-492d-9b9d-4fbfb54b1fb7)

## Type of change

- [x] Bug fix (fixes an issue)
- [ ] New feature (adds functionality)
- [ ] Improvement (improves codebase without changing functionality)

### Development

- [x] The changelog has been updated as appropriate
- [x] Lint rules pass locally
- [ ] Application changes have been tested thoroughly
- [ ] Automated tests covering modified code pass

### Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### Code review

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [ ] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

### CODE REVIEW

File: src/components/ReportsPage/HeatmapPage/Tables/LevelsDrivenTable.tsx [hunk 1]
```diff
@@ -77,13 +84,6 @@ export const LevelsDrivenTable = ({
                   />
                 </TableCell>
               )}
-              {notGroupedByServices && (
```
Comment (FoodProduct) [inline src/components/ReportsPage/HeatmapPage/Tables/LevelsDrivenTable.tsx:80]: Is this change intentional? It doesn't match the PR or changelog description

Comment (PavelZurek) [reply src/components/ReportsPage/HeatmapPage/Tables/LevelsDrivenTable.tsx:80]: Yes, sorry, I moved this to just remove the unnecessary condition here.

**Review by FoodProduct — APPROVED** [review]
